.pre-commit-config.yaml

```diff
@@ -23,4 +23,7 @@ repos:
     hooks:
       - id: ruff
         args: [--fix]
-      - id: ruff-format
+  - repo: https://github.com/djlint/djLint
+    rev: v1.34.1
+    hooks:
+      - id: djlint-django
```

Comment:
@JisanAR03 any reasons to avoid using `- id: djlint-reformat-django`?

Reply:
@JisanAR03 any reason for this? Was there an error?

Reply:
@DonnieBLT , yeah sir, there are a lot of error when we use this one and even there isn't any clues about the error 
![image](https://github.com/OWASP/BLT/assets/97744811/fcc0fccb-d02b-478f-9ffa-b18afb158c4b)

Reply:
This looks like a reformatting suggestion (not an error). 
Could you add, test and submit the suggested changes?

Reply:
I tried all of this, and the Lint service solved all the errors. and when we use reformation, it detects errors, but when I check the code, there is nothing wrong🙂

Reply:
This is how pre-commit wants the code to look like in terms of readability.

Your steps to resolve here would be following:
  - add (keep) `- id: djlint-reformat-django` option
  - run `pre-commit run -a`
  - inspect changes, test the code works (no breaking changes introduced by reformatting)
  - add all changes (as a single commit)
  - run `pre-commit run -a` (again)
  - make sure no new changes being generated
  - if there are new changes -- repeat the process starting step #2
  - when pre-commit is satisfied, commit any uncommited stuff and push it to your PR

Reply:
ok I will check this out

Reply:
The pre-commit checks are vital for keeping the code base quality at a certain level.
You did a great work while implementing djlint part of it! Thank you!

Reply:
Yes, nice work @JisanAR03 🎖️🎖️🎖️🎖️🎖️🎖️🎖️🎖️🎖️🎖️

Reply:
@DonnieBLT sir , now implemented "djlint-reformat-django". @arkid15r thank you so much for you suggestion😊